ios/Sources/Model/FeedItem.swift

```diff
@@ -0,0 +1,51 @@
+import DroidKaigiMPP
+
+public enum FeedItemType: Equatable {
+    case blog(Blog)
+    case podcast(Podcast)
+    case video(Video)
+}
+
+public struct FeedItem: Equatable, Identifiable {
+    public var id: String
+    public var image: Image
+    public var link: String
+    public var media: Media
+    public var publishedAt: Date
+    public var summary: MultiLangText
+    public var title: MultiLangText
+    public var publishedDateString: String?
+
+    public init(
+        id: String,
+        image: Image,
+        link: String,
+        media: Media,
+        publishedAt: Date,
+        summary: MultiLangText,
+        title: MultiLangText
+    ) {
+        self.id = id
+        self.image = image
+        self.link = link
+        self.media = media
+        self.publishedAt = publishedAt
+        self.summary = summary
+        self.title = title
+        self.publishedDateString = nil
+    }
+
+    public init(from model: DroidKaigiMPP.FeedItem) {
+        self.id = model.id
+        self.image = Image(from: model.image)
+        self.link = model.link
+        self.media = Media.from(model.media)
+        self.publishedAt = Date(
+            timeIntervalSince1970: Double(model.publishedAt.toEpochMilliseconds())
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg)
May be a better choice to [export](https://kotlinlang.org/docs/mpp-build-native-binaries.html#export-dependencies-to-binaries) kotlinx-datetime since it has [`Conversations`](https://github.com/Kotlin/kotlinx-datetime/blob/master/core/darwin/src/Converters.kt) which can help us to keep the consistency when we do something similar in `iosMain`?

Reply:
I'm not sure,

Does `export kotlinx-datetime` mean creating a converting function in KMM module that can be used in Swift? For example, `Instant.toNSDate()` can be used in Swift.

Reply:
> Does export kotlinx-datetime mean creating a converting function in KMM module that can be used in Swift? For example, Instant.toNSDate() can be used in Swift.

Yes! But strictly speaking, it's not creating but export those functions.

Reply:
Sorry for late replying 🙇🏼 

Can you do it another PR after this PR merged?
I don't know how to do it 😭